oci-seccomp-bpf-hook.go

```diff
@@ -59,15 +59,15 @@ BPF_HASH(parent_namespace, u64, unsigned int);
 BPF_PERF_OUTPUT(events);
 
 // data_t used to store the data received from the event
-struct syscall_data
-{
-	// PID of the process
-	u32 pid;
-	// the syscall number
-	u32 id;
-	// command which is making the syscall
-	char comm[16];
-	int exit;
+struct syscall_data {
+    // PID of the process
+    u32 pid;
+    // the syscall number
+    u32 id;
+    // command which is making the syscall
+    char comm[16];
+    // a flag to check if the parent process is running
+    int running;
```

Comment:
I don't like the naming and description of this field. It's a bit misleading as when we're setting it to 1, the "parent" is actually the current process.

I suggest calling `stopTracing` and make it a bool. This seems more straight forward and easier to document (i.e., when are we stopping to trace?).